src/js/components/picklist/index.js

```diff
@@ -58,7 +58,7 @@ const PicklistEmpty = View.extend({
 const PicklistItem = View.extend({
   tagName: 'li',
   itemTemplate: hbs`
-    <div class="flex-grow">{{#if icon}}{{fa icon.type icon.icon classes=icon.classes}}{{/if}}<span>{{matchText text query}}</span></div
+    <div class="flex-grow">{{#if icon}}{{fa icon.type icon.icon classes=icon.classes}}{{/if}}<span>{{matchText text query}}</span></div>
```

Comment:
Not related to removing the print button. But snuck it in on this PR.

Just a missing closing tag from the [Save + Go Back PR](https://github.com/RoundingWell/care-ops-frontend/pull/806).